docs/rest-api/rest-notebook.md

```diff
@@ -433,6 +433,58 @@ limitations under the License.
   <table class="table-configuration">
     <col width="200">
     <tr>
+      <th>Run paragraph synchronously</th>
+      <th></th>
+    </tr>
+    <tr>
+      <td>Description</td>
+      <td>This ```POST``` method runs the paragraph synchronously by given notebook and paragraph id. This API can return SUCCESS or ERROR depending on the outcome of the paragraph execution
+      </td>
+    </tr>
+    <tr>
+      <td>URL</td>
+      <td>```http://[zeppelin-server]:[zeppelin-port]/api/notebook/run/[notebookId]/[paragraphId]```</td>
+    </tr>
+    <tr>
+      <td>Success code</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <td> Fail code</td>
+      <td> 500 </td>
+    </tr>
+    <tr>
+      <td> sample JSON input (optional, only needed when if you want to update dynamic form's value) </td>
+      <td><pre>
+{
+  "name": "name of new notebook",
+  "params": {
+    "formLabel1": "value1",
+    "formLabel2": "value2"
+  }
+}
+      </pre></td>
+    </tr>
+    <tr>
+      <td> sample JSON response </td>
+      <td><pre>
+{
+   "status": "OK",
+   "body": {
+       "code": "SUCCESS",
+       "type": "TEXT",
+       "msg": "Current time = 16:14:18\n"
```

Comment:
The line return is the result of the shell command output. I have tested with a Spark code ( `println("now " + new java.util.Date())` and I also get the `\n` at the end of the text stream. It's the normal behavior. We don't see the line return when the text is rendered inside a `<div>`